### ISSUE TITLE
In small logs, top-right buttons can obscure text

### ISSUE THREAD

**Issue by mitchhentges** [taskcluster/taskcluster-web#401]
**Summary:**
[Example log](https://taskcluster-web.netlify.com/tasks/Zq9m4s9qQsqFYPfVMYJkkw/runs/9/logs/https%3A%2F%2Fqueue.taskcluster.net%2Fv1%2Ftask%2FZq9m4s9qQsqFYPfVMYJkkw%2Fruns%2F9%2Fartifacts%2Fpublic%2Flogs%2Flive_backing.log) (what's the text after `signing_task_schema`?). In cases where the log can't scroll, the buttons in the top-right obscure text. The only workaround is to view the raw log, which isn't ideal.

**Comment by helfi92** [comment]
@mitchhentges Agreed, this is not ideal. Any thoughts on how we can make this better?

**Comment by mitchhentges** [comment]
I think there's two main options here:
1. Have these hide/show based on if the user hovers in the top-right of their screen. Downside to this is that discover-ability is lowered, and it's possible for the hide/show effect to be annoying
2. Dedicate a portion of the screen to be dedicated to buttons and don't overlap buttons with the logs. The downside to this is that there's less room for logs. I personally prefer this behaviour, because I <3 consistency (width of visible lines is the same throughout the log, nothing is conditionally taking the log's space) rather than having extra space with conditional overlap

**Comment by helfi92** [comment]
Most views have their actions fixed to the bottom right. We could perhaps add them there and increase the number of empty lines at the end of the log to make sure the bottom log can always be read.

**Comment by mitchhentges** [comment]
That sounds phenomenal, good call :+1:

**Comment by varundey** [comment]
I would love to work on this :)

**Comment by helfi92** [comment]
Go for it.

**Comment by varundey** [comment]
The way I see it, I can add `padding-bottom` to html class with value equal to button height. What do you think?

**Comment by helfi92** [comment]
The component has a property you can use to add new lines. We already add 1 line. All you have to do is increase that number.

**Comment by varundey** [comment]
Okay here is the output after changing increasing the number to 3. I fear it might still overlap if the line length is more
<img width="1205" alt="screen shot 2019-01-21 at 12 29 41 am" src="https://user-images.githubusercontent.com/9202135/51443771-fe53ce00-1d13-11e9-9f65-4345fda19658.png">

**Comment by helfi92** [comment]
This looks good :) Feel free to open a pull-request.

**Comment by helfi92** [comment]
Yes.
